spec/controllers/admin/emergency_message_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe Admin::EmergencyMessagesController do
+  before :each do
+    @user = create!(:user, name: 'hello', admin: true, email: 'lol@biz.info', password: 'irrelevant')
+    sign_in @user
+    EmergencyMessage.destroy_all()
+    @message = create(:emergency_message, :message => "Test message", :show => false)
+  end
+
+  it "displays the emergency message screen" do
+    get :edit, {:id => '1'}
+    response.should render_template('admin/emergency_messages/edit')
+    response.should be_success
+  end
+
+  describe "#update" do
+    let(:params){ {id: @message.id, message: { name: 'Emergency message', show: true }} }
+
+    context "that works" do
+      before{
+        EmergencyMessage.any_instance.stub(update_attributes: true)
+      }
+      it "purges the cache" do
+        controller.should_receive(:purge_all_pages)
```

Comment:
now that purge_all_pages is gone from the PR, it would be great if you could also update this spec.